src/org/zaproxy/zap/extension/quickstart/QuickStartPanel.java

```diff
@@ -283,6 +299,20 @@ public void actionPerformed(java.awt.event.ActionEvent e) {
 		return confButton;
 	}
 
+	public void updateConfButton(boolean apiState) {
+		if (confButton == null) {
+			getConfButton();
+			return;
+		}
+		//PnH button has the same enable state as the API
+		confButton.setEnabled(apiState);
+		if (apiState) {
+			confButton.setToolTipText(Constant.messages.getString("quickstart.button.tooltip.mitm"));
+		} else {
+			confButton.setToolTipText(Constant.messages.getString("quickstart.mitm.api.disabled"));

```

Comment:
It isn't a Toggle Button though, it's not a Binary on/off. It's a press to activate.... if I use a Toggle Button after the user clicks it once, the browser opens and it stays "selected"....